### DISCUSSION TITLE
Scalar Quantization memery cost estimation

### DISCUSSION THREAD

**Discussion by xtyDoge** [qdrant/qdrant#4803] [Q&A]
I have a 3 nodes qdrant cluster, and a collection with 16million vectors, and it has 2 replicas, the vector config shows below 
```
        "vectors": {
          "size": 512,
          "distance": "Dot",
          "on_disk": true
        },
```
I want to apply Scalar Quantization on this collection to reduce disk loading when querying this collection. How can I estimate memory cost on each node? 
The result I get is 16,000,000 * 512 * 4B(float64) / 4(Quantization compress factor), approxiamtely total 8GB, on each node is 8GB * 2 /3 = 5.3G memory usage, is that right?

**Comment by timvisee** [comment]
Roughly, yes. Note that its float32, which is indeed 4 bytes.

That would just be the quantized vectors though. It does not include other memory consumers, such as payload data and a bit of overhead for the collection itself.

As always, I'd recommend you to test in practice whether your above estimation holds true.
